### PR TITLE
Fixes #1031 Punjabi, Nepali Language Added

### LINKED ISSUE #1031: Extend Language Support
**Actual Behaviour**
There are languages, but more should be added

**Expected Behaviour**
Integration for other languages

**Steps to reproduce it**
- Add more language files

**Would you like to work on the issue?**
Yes !

### PR DESCRIPTION
**Pull request by YashKumarVerma**
adding to list

Fixes #1031 

Changes: 
- Translated and added two new language files
- Integrated 2 languages to the settings tab

Demo Link: https://pr-1045-fossasia-susi-web-chat.surge.sh

image

### CODE REVIEW

File: src/components/Translate/np.json [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  "Expand your experience, get closer, and stay current":"आफ्नो अनुभव विस्तार गर्न र विस्तारित रहनुहोस्",
+  "Add your phone number":"आफ्नो फोन नम्बर थप्नुहोस्",
+  "We will text a verification code to this number. Standard SMS fees may apply.":"हामी यस नम्बरमा एक प्रमाणिकरण कोड पाठ गर्नेछौं। मानक एसएमएस फीस लागू हुन सक्छ।",
+  "Save Changes":"परिवर्तनहरू सुरक्षित गर्नुहोस",
+  "Account":"खाता",
+  "Theme":"विषयवस्तु",
+  "Your Account":"तपाईंको खाता",
+  "Your Email":"तिम्रो इमेल",
+  "Country/region :"देश / क्षेत्र",
+  "Phone number :"फोन नम्बर",
```
Comment (isuruAb) [inline src/components/Translate/np.json]: you missed one ' " ' here

**Comment by YashKumarVerma** [comment]
@fossasia/susi-ai

**Comment by YashKumarVerma** [comment]
@mariobehling

**Comment by YashKumarVerma** [comment]
@nileshgulia1 I cannot see these commits in the rebase window, what should i do ? please help

**Comment by nileshgulia1** [comment]
@YashKumarVerma  It is always recommended that you work on a feature branch and do a interactive rebase before making a PR.First rebase your code with upstream master and resolve conflicts (if any).

**Comment by YashKumarVerma** [comment]
@nileshgulia1 can you please help me squashing these ?

**Comment by nileshgulia1** [comment]
First you need to identify the last commit before the ones you want to merge, which is #86b6320 in your case. Run``` git log``` and remember the hash of last commit (ie. #86b6320)

1.Run ```git rebase -i  #86b6320```, this will bring up your default text editor
2.Keep the first commit as ```pick```, and change all the other pick to ```squash``` (or s for short).
3.Now save and quit the text editor, the rebase will run until the end. You might meet conflicts like you do in rebasing. Fix them first and continue to rebase.
4.Then commit with the new message and use```git push -f```

**Comment by abishekvashok** [comment]
@YashKumarVerma the last step should be `git push -f`

**Pull request merged**